docs/getting-started/quickstart.md

```diff
@@ -35,10 +26,98 @@ export NVM_DIR="$HOME/.nvm"
 nvm install 20
 nvm use 20
    ```
-<h3> Installing Postgres </h3>
+</TabItem>
+<TabItem value="win-ps" label="Windows">
+
+Download Node.js 20 or later from the [official Node.js download page](https://nodejs.org/en/download) and install it.
+After installing Node.js, create the following folder: `C:\Users\%user%\AppData\Roaming\npm`
+(`%user%` is the Windows user on which you are logged in).
+</TabItem>
+</Tabs>
+
+### 2. Create the application folder
```

Comment:
```suggestion
### 2. Create an application
```

Reply:
@qianl15 here I think it's important to introduce "the application folder" in big bold letters. It may be obvious to some users but for others, it is important to establish "the application lives in a specific named folder". We then refer back to it several times below when we say things like "run this from the application folder." So having the word "folder" in the heading makes it a lot more memorable. Does that make sense? Let me know what you think?

Reply:
Makes sense